### DISCUSSION TITLE
How to know that the last instance has already been stored?

### DISCUSSION THREAD

**Discussion by aleja-lopez** [dcm4che/dcm4chee-arc-light#3431] [Q&A]
Hello everyone, I am working on a script that is in charge of alerting each time a new series is stored, I would like to know if DICOM WEB has any parameter that tells me when the last instance of a series has been stored.

**Comment by vrindanayak** [comment]
There is no way to check for completeness of a study / series received by the archive i.e. additional objects belonging to this study / series shall be stored or not. Normally, objects of a series are sent together and not separately. However, a user may choose to upload bulkdata files or choose to copy/move objects from one study / series to another, in which case the objects of a previously sent series gets incremented.

Archive has functions whereby it can notify other systems on receive of objects. See
- [Instance Availability Notification](https://github.com/dcm4che/dcm4chee-arc-light/wiki/Instance-Availability-Notification)
- [Synchronizing external HL7 receivers](https://github.com/dcm4che/dcm4chee-arc-light/wiki/Synchronize-external-HL7-Receivers) on [receive of studies](https://github.com/dcm4che/dcm4chee-arc-light/wiki/Received-Studies)
- [Forwarding of received instances](https://github.com/dcm4che/dcm4chee-arc-light/wiki/Forwarding-of-received-instances)
